references/classification/train.py

```diff
@@ -28,7 +30,7 @@ def target_transform(target):
     return target.unsqueeze(dim=0)
 
 
-def load_data(traindir, valdir, img_size=224, crop_pct=0.8):
+def load_data(data_path, use_OpenFire=False, download=False, img_size=224, crop_pct=0.8):
```

Comment:
You can remove `download=False` and perhaps replace `user_openfire` all lowercase :ok_hand: